straxen/rundb.py

```diff
@@ -206,16 +200,16 @@ def _find(self, key: strax.DataKey,
                         'host': self.hostname,
                         'type': key.data_type,
                         'protocol': strax.FileSytemBackend.__name__,
-                        # TODO: duplication with metadata stuff elsewhere?
-                        'meta': {'lineage': key.lineage}
+                        'did': key_to_rucio_did(key),
+                        'lineage_hash': key.lineage_hash,
```

Comment:
Ah I think this answers my question above. So bootstrax is using this frontend and thus this line sets the `lineage_hash` to be a top level field. Is that right?

Reply:
yes, so instead of doing it at bootstrax do this already at the start in the writing part of the frontend. In contrast to the other fields, you already know this up front so bootstrax does not have to upload it later after we finished writing